app/Exceptions/Handlers/NoEncryptionKey.php

```diff
@@ -37,7 +40,11 @@ public function check(HttpException $e): bool
 	public function renderHttpException(SymfonyResponse $defaultResponse, HttpException $e): SymfonyResponse
 	{
 		try {
-			touch(base_path('.NO_SECURE_KEY'));
+			try {
+				touch(base_path('.NO_SECURE_KEY'));
+			} catch (Throwable) {
+				// do nothing
+			}
```

Comment:
I pushed a commit https://github.com/LycheeOrg/Lychee/pull/1495/commits/e055fe2839368d51cdf5de192ec0fd3628b009f7 which removes the need for this file `NO_SECURE_KEY` completely. I have never liked the approach which uses an extra file, because it may get out-of-sync with the actual installation state. Now even more, if we cannot even guarantee that the file is created reliably.

Instead https://github.com/LycheeOrg/Lychee/pull/1495/commits/e055fe2839368d51cdf5de192ec0fd3628b009f7 directly checks for the existence of the application key.